### PR TITLE
Complete README

### PR DESCRIPTION
**Pull request by biocdgv**
Add some changes to Metabiome's README:

- Add badges
- Add binning section.
- Add the following sections: basic usage, ask for help, contribution and acknowledgments.

Also, remove unnecessary "" in Kraken2 usage function output.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,107 @@
...
+- CoverM
+
+#### Bins refinement and assessment
+- DAS Tool
+- CheckM
+
 ## Pipeline
-![Pipeline](https://i.imgur.com/NcpxAXI.png)
+![Pipeline](https://i.imgur.com/ZpCIXYV.png)
+)
```
Comment (Nesper94) [inline README.md:49]: Remover paréntesis que sobra:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1,34 +1,107 @@
...
+order to avoid duplications.
+
+## Ask for help
+
+If you have questions about how to use `Metabiome`, or are interested in
+active collaboration of this project, do not hesitate to contact:
+
+Cristian Grisales-Vargas: cristian.grisales@udea.edu.co
+Estefany Lorenzana: estefany.lorenzana@udea.edu.co
+Juan Camilo Arboleda: juan.arboleda2@udea.edu.co
```
Comment (Nesper94) [inline README.md:102]:
```suggestion
[Cristian Grisales-Vargas](mailto:cristian.grisales@udea.edu.co)  
[Estefany Lorenzana](mailto:estefany.lorenzana@udea.edu.co)  
[Juan Camilo Arboleda](mailto:juan.arboleda2@udea.edu.co)  
```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,107 @@
...
-cd metabiome/
-bash install.sh
-```
-The installation script will create Conda environments with all the software you
-can use with Metabiome.
+## About Metabiome
+
+`Metabiome` is a bioinformatic pipeline for microbiome analysis. It consists of
+a series of wrapper scripts written in Bash. It is also contained in Conda
+environments to ease its use. `Metabiome` includes several features of common
```
Comment (Nesper94) [inline README.md:13]:
```suggestion
environments to ease its use. Metabiome includes several features of common
```

File: README.md [hunk 4]
```diff
@@ -1,34 +1,107 @@
...
+## Installation
+
+Download or clone this repository and run `install.sh` in the command line:
+
+```bash
+cd metabiome/
+bash install.sh
+```
+The installation script will create Conda environments with all the software you
+can use with `Metabiome`.
```
Comment (Nesper94) [inline README.md:68]:
```suggestion
can use with Metabiome.
```

File: README.md [hunk 5]
```diff
@@ -1,34 +1,107 @@
...
+
+
+## Basic usage
+
+`Metabiome` can be invoked like so:
```
Comment (Nesper94) [inline README.md:73]:
```suggestion
Metabiome can be invoked like so:
```

File: README.md [hunk 6]
```diff
@@ -1,34 +1,107 @@
...
+```bash
+metabiome --help
+```
+See the  [documentation](https://metabiome.readthedocs.io/en/latest/) for
+further details.
+
+
+## Contribute to Metabiome
+
+If you want to contribute to `Metabiome`, you should fork the project.
```
Comment (Nesper94) [inline README.md:84]:
```suggestion
If you want to contribute to Metabiome, you should fork the project.
```

File: README.md [hunk 7]
```diff
@@ -1,34 +1,107 @@
...
+Then, make your desired changes and submit a pull request in order to be
+evaluated by the current developers.
+
+## Submit an issue
+
+If you have problems or suggestions regarding `Metabiome`, please submit an
```
Comment (Nesper94) [inline README.md:90]:
```suggestion
If you have problems or suggestions regarding Metabiome, please submit an
```

File: README.md [hunk 8]
```diff
@@ -1,34 +1,107 @@
...
+issue on [Metabiome issues](https://github.com/Nesper94/Metabiome/issues).
+Also, please search the previous issues (if any) before submiting a new one in
+order to avoid duplications.
+
+## Ask for help
+
+If you have questions about how to use `Metabiome`, or are interested in
```
Comment (Nesper94) [inline README.md:97]:
```suggestion
If you have questions about how to use Metabiome, or are interested in
```

File: README.md [hunk 9]
```diff
@@ -1,34 +1,107 @@
...
+- CheckM
+
 ## Pipeline
-![Pipeline](https://i.imgur.com/NcpxAXI.png)
+![Pipeline](https://i.imgur.com/ZpCIXYV.png)
+)
+
+## Prerequisites for running Metabiome
+
+There are two prerequisites for running `Metabiome`:
```
Comment (Nesper94) [inline README.md:53]:
```suggestion
There are two prerequisites for running Metabiome:
```

**Review by Nesper94 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Estefany-Lorenzana) [reply README.md:102]: It looks better with the link!

File: README.md [hunk 10]
```diff
@@ -1,34 +1,89 @@
...
 
+[![License: MIT](https://img.shields.io/badge/License-MIT-orange.svg)](https://github.com/Nesper94/Metabiome/blob/master/LICENSE)
+[![Maintained?](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/Nesper94/Metabiome/pulse)
+[![Made with: bash](https://img.shields.io/badge/Made%20with-Bash-1f425f.svg)](https://www.gnu.org/software/bash/)
+[![Read: the docs](https://img.shields.io/badge/read-the%20docs-blue)](https://metabiome.readthedocs.io/en/latest/)
+
+
+<p align="left">
+  <img width = "500" src="https://i.imgur.com/FGbY9Ed.png"/>
+</p>
```
Comment (Nesper94) [inline README.md:12]: Put logo outside paragraph and next to the title:
```suggestion
<img src="https://i.imgur.com/FGbY9Ed.png" width=120 align="left" />

# Metabiome
A flexible and modular pipeline for microbiome analysis.

[![License: MIT](https://img.shields.io/badge/License-MIT-orange.svg)](https://github.com/Nesper94/Metabiome/blob/master/LICENSE)
[![Maintained?](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/Nesper94/Metabiome/pulse)
[![Made with: bash](https://img.shields.io/badge/Made%20with-Bash-1f425f.svg)](https://www.gnu.org/software/bash/)
[![Read: the docs](https://img.shields.io/badge/read-the%20docs-blue)](https://metabiome.readthedocs.io/en/latest/)
```

**Review by Nesper94 — CHANGES_REQUESTED** [review]

**Review by Estefany-Lorenzana — DISMISSED** [review]

**Review by Nesper94 — APPROVED** [review]

**Review by Estefany-Lorenzana — APPROVED** [review]

**Pull request merged**